prefs.js

```diff
@@ -54,15 +56,33 @@ function buildPrefsWidget() {
         }),
         visible: true
     });
-    prefsWidget.attach(leftPaddingEntry, 1, 1, 1, 1);
+    prefsWidget.attach(leftPaddingEntry, 1, index, 1, 1);
+
+    /* right-padding */
+    let rightPaddingLabel = new Gtk.Label({
+        label: 'Right padding:',
+        halign: Gtk.Align.START,
+        visible: true
+    });
+    prefsWidget.attach(rightPaddingLabel, 0, ++index, 1, 1);
+
+    let rightPaddingEntry = new Gtk.SpinButton({
+        adjustment: new Gtk.Adjustment({
+            lower: 0,
+            upper: 100,
+            step_increment: 1
+        }),
+        visible: true
+    });
+    prefsWidget.attach(rightPaddingEntry, 1, index, 1, 1);
```

Comment:
For some reason I'm not very fond of this `index` variable change (and I'm not very fond of the GNOME implementation in the first place :sweat_smile: ). How about this, grouping the initializations, doing the index++ separatey and the attaching the elements:
```
    /* right-padding */
    let rightPaddingLabel = new Gtk.Label({
        label: 'Right padding:',
        halign: Gtk.Align.START,
        visible: true
    });
    
    let rightPaddingEntry = new Gtk.SpinButton({
        adjustment: new Gtk.Adjustment({
            lower: 0,
            upper: 100,
            step_increment: 1
        }),
        visible: true
    });

    index++;
    prefsWidget.attach(rightPaddingLabel, 0, index, 1, 1);
    prefsWidget.attach(rightPaddingEntry, 1, index, 1, 1);
```

I'm aware that this has been here since before your PR, but I figured we could kill two birds with one stone :)